src/menus/position_settings_menu.cpp

```diff
@@ -90,8 +90,7 @@ void PosSettingsMenu::render(Font& font) {
     if (button_is_down(Controller::DPAD_RIGHT) ||
         button_is_down(Controller::DPAD_LEFT) ||
         button_is_down(Controller::DPAD_UP) ||
-        button_is_down(Controller::DPAD_DOWN))
-    {
+        button_is_down(Controller::DPAD_DOWN)) {
```

Comment:
This new line helps readability, because without it, it would be hard at a first glance to make appart what is in the condition and where the body starts.